.dockerignore

```diff
@@ -0,0 +1,5 @@
+*
```

Comment:
better add everything and ignore:

/node_modules
/dist

Reply:
Ok, same as boldo-server, I will use the .gitignore as a template but without the .env file.